sysboundary/sysboundary.cpp

```diff
@@ -531,9 +380,9 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg::C
       }
    }
 
-   // The following is done so that everyone knows their neighbour's
-   // layer flags.  This is needed for the correct use of the system
-   // boundary local communication patterns.
+   // The following is done so that everyone knows their neighbour's layer
+   // flags.  This is needed for the correct use of the boundary local
```

Comment:
Oops! Here it actually does mean system boundary local communication patterns. Exactly the difference between system boundaries and process boundaries!